Core/OxidEeEvents.php

```diff
@@ -72,6 +72,8 @@ public static function extendPaymentMethodTable()
             'WDOXIDEE_CREDITORID' => "varchar(35) default '' NOT NULL",
             'WDOXIDEE_SEPAMANDATECUSTOM' => "text default '' NOT NULL",
             'WDOXIDEE_SEPAMANDATECUSTOM_1' => "text default '' NOT NULL",
+            'WDOXIDEE_SEPAMANDATECUSTOM_2' => "text default '' NOT NULL",
+            'WDOXIDEE_SEPAMANDATECUSTOM_3' => "text default '' NOT NULL",
```

Comment:
Where is this one being used? as in the SQL is see an update only on WDOXIDEE_SEPAMANDATECUSTOM and WDOXIDEE_SEPAMANDATECUSTOM_1

Reply:
Those 2 fields get used when you add 2 more languages. We missed that before. But from 4 languages on, adding a fifth, Oxid will add `WDOXIDEE_SEPAMANDATECUSTOM_5`, and so on, dynamically.